priv/data/flows/geolocation.json

```diff
@@ -0,0 +1,329 @@
+                                    "name": "Failure",
+                                    "exit_uuid": "a2b682cc-7f71-40b5-b0e3-5fedd3e2ea19"
+                                }
+                            ],
+                            "cases": [
+                                {
+                                    "uuid": "c8b1e374-aaa2-47b3-b15c-4bd4546c16ef",
+                                    "type": "has_only_text",
+                                    "category_uuid": "e73cc142-02c3-41c0-b26a-5d354301f5a5",
+                                    "arguments": [
+                                        "Success"
+                                    ]
+                                }
+                            ]
+                        },
+                        "exits": [
+                            {
+                                "uuid": "625519e4-c868-4e5e-af75-0e0d0ee11ce6",
+                                "destination_uuid": "d82c194d-c537-45ba-8fa0-0b1e0614b657"
+                            },
+                            {
+                                "uuid": "a2b682cc-7f71-40b5-b0e3-5fedd3e2ea19",
+                                "destination_uuid": "687cfea4-6e8b-44a3-9d9d-a368e57c7ebb"
+                            }
+                        ],
+                        "actions": [
+                            {
+                                "uuid": "9ab60aed-40b2-4c02-a03b-f39404281b03",
+                                "url": "geolocation",
+                                "type": "call_webhook",
+                                "result_name": "loc",
+                                "method": "FUNCTION",
+                                "headers": {
+                                    "Content-Type": "application/json",
+                                    "Accept": "application/json"
+                                },
+                                "body": "{\n \"lat\": \"@results.result_1.latitude\",\n \"long\": \"@results.result_1.longitude\"\n}"
+                            }
+                        ]
+                    },
+                    {
+                        "uuid": "d82c194d-c537-45ba-8fa0-0b1e0614b657",
+                        "exits": [
+                            {
+                                "uuid": "66a9775d-cf7f-4e72-8aaf-d288fa743a47",
+                                "destination_uuid": null
+                            }
+                        ],
+                        "actions": [
+                            {
+                                "uuid": "f7c5c029-0c47-40c3-af49-69dcc6430bf1",
+                                "type": "send_msg",
+                                "text": "Location details\n\nWard - @results.loc.ward\nState - @results.loc.state\nPostal Code - @results.loc.postal_code\nDistrict - @results.loc.district\nCity - @results.loc.city\nCountry - @results.loc.country\nFull address - @results.loc.address",
```

Comment:
We have removed `ward` from the location results. So can we remove that from this flow too?
https://github.com/glific/glific/pull/3814